services/core-api/app/api/incidents/response_models.py

```diff
@@ -49,12 +47,18 @@
     }
 )
 
+MINE_INCIDENT_CATEGORY_XREF_MODEL = api.model(
```

Comment:
this will also get hidden when you use the relationship with the secondary flag to get many-to-many.

Reply:
So this MINE_INCIDENT_CATEGORY_XREF_MODEL is not required if I make those changes you mentioned in your [other comment](https://github.com/bcgov/mds/pull/1116#discussion_r361201547)?

Reply:
correct, as far the caller of our API is concerned, a mine_incident has a set of categories, that's how it should look. having a set of xrefs would not be ideal.